docs/source/setup/benchmarking.rst

```diff
@@ -0,0 +1,33 @@
+Benchmarking
+============
+
+To run a benchmark, one can use the command line interface ``PyVRP`` provides.
+After installation, it is available as ``pyvrp`` from the command line.
```

Comment:
Does this hold both for building from sources and installing through PyPI? In the first case, would it be helpful to mention Poetry environment must be activated or you must run `poetry run pyvrp`? Finally, this requires `benchmark` group dependencies, which are optional, will they be installed by default via PyPI?

Reply:
For developing locally, this needs to run within the poetry environment, but we'll make that clear in the contributing/developing sections. The command is provided when the package is installed, and that requires the benchmark group which I will add to the core dependencies now.